source/_components/rotel.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: "Rotel Media Player"
+description: "Rotel Media Player platform implementation."
+date: 2019-04-27 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rotel.svg
+ha_release: 0.93
+ha_category: Media Player
+ha_iot_class: Local Push
+---
+
+A Media Player platform that can control a Rotel Amplifier or Processor.
+
+At the moment the platform only supports the Rotel RSP-1570 processor but the underlying third party library could be adapted to support other devices of the same vintage.
+
+## Supported Devices
+
+* [Rotel RSP-1570 Processor](http://www.rotel.com/en-gb/product/rsp-1570)
+
+## {% linkable_title Configuration %}
+
+Example configuration.yaml entry
+
+```yaml
+media_player:
+- platform: rotel
+  device: /dev/ttyUSB0
+  source_aliases:
+    TUNER:
+    TAPE:
+    MULTI:
+    VIDEO 1: CATV
+    VIDEO 2: NMT
+    VIDEO 3: APPLE TV
+    VIDEO 4: FIRE TV
+    VIDEO 5: BLU RAY
+```
+
+{% configuration %}
+device:
+  description: Connection string for the device. See below for details.
+  required: true
+  type: string
+name:
+  description: Name of the device.
+  required: false
+  default: Rotel RSP-1570
+  type: string
+source_aliases:
+  description: Mapping of default sources to aliases. The default sources are `' CD'`, `TAPE`, `TUNER`, `MULTI`, `VIDEO 1`, `VIDEO 2`, `VIDEO 3`, `VIDEO 4`, `VIDEO 5`. Each key must exactly match a default source name. The values must exactly match the labels configured in the device.  An empty alias indicates that the source is not used. A missing alias means that the default source should appear as-is in the source list.
+  required: false
+  default: {}
+  type: dict
```

Comment:
`dict` is not a valid type for our documentation. Please see:

<https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration>